compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -2009,15 +2010,19 @@ impl<'tcx> Debug for Rvalue<'tcx> {
             }
             Len(ref a) => write!(fmt, "Len({:?})", a),
             Cast(ref kind, ref place, ref ty) => {
-                write!(fmt, "{:?} as {:?} ({:?})", place, ty, kind)
+                let ty = ty::print::with_no_trimmed_paths!(format!("{ty}"));
```

Comment:
yeah, i didnt do this because I was worried that the other things getting formatted might be affected by the "no trimmed paths". didnt actually check to see though